pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+		xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>BBFW</groupId>
+	<artifactId>traffic_simulation</artifactId>
+	<version>1.0-SNAPSHOT</version>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<kotlin.version>1.1.2</kotlin.version>
+		<main.class>BBFW-Project</main.class>
```

Comment:
Das muss der Name der Klasse sein, die die `main` Methode enthält, sonst kann man die erstellte JAR nicht mit `java -jar ${path/to/JAR/file}` ausführen.